cspatients/util.py

```diff
@@ -94,19 +91,20 @@ def save_model_changes(post_data):
     if patientserializer.is_valid():
         patientserializer.save()
     if patiententryserializer.is_valid():
-        updated_patiententry = patiententryserializer.save()
-    return updated_patiententry
+        return patiententryserializer.save()
+    else:
+        return patiententry
```

Comment:
If the input isn't valid you might want to return the errors (these can be gotten from the serializer) so that we can show an informative error message